mono/metadata/class.c

```diff
@@ -688,16 +717,38 @@ inflate_generic_type (MonoImage *image, MonoType *type, MonoGenericContext *cont
 			return NULL;			
 		}
 
+#ifdef DEBUG_INFLATE_CMODS
```

Comment:
Can we hook this up to some autogen setting or document where it's set? Does someone just add a #define somewhere? Could you add a commented-out copy of that define, so people know where to put it?

Reply:
There's an `#undef DEBUG_INFLATE_CMODS` at the top of the file:
https://github.com/mono/mono/pull/13320/files#diff-f085caaf39e861140ff40b9ef0c516d9R76